.lintstagedrc

```diff
@@ -2,10 +2,8 @@
 	"**/*.js": [
 		"prettier --ignore-path .gitignore --write",
 		"yarn lint --fix",
-		"git add"
```

Comment:
thanks for this fix. `.lintstagedrc` should be a JSON, so please remove the trailing commas that were left there (after yarn lint--fix and prettier --write)